src/shared/components/form/formSelect/formSelect.js

```diff
@@ -2,6 +2,18 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 export default class FormSelect extends Component {
+  onChange(e) {
+    // make sure we have a valid value before propogating the on change call
+    let isValid = true;
+    if (this.props.validationFunc) {
```

Comment:
Is this ever rendered without the `validationFunc` prop? What scenario did you run into where this was undefined?

Reply:
Yes, we may choose to use the `FormSelect` component without defining a `validationFunc` in any case we don't need validation or the component is not required. In those cases the `validationFunc` will be undefined.